### PR TITLE
fixed follow btn in FriendsRecommendationList

### PR DESCRIPTION
**Pull request by IhorKalyta**

### CODE REVIEW

File: client/src/components/FriendsRecommendationsList/index.tsx [hunk 1]
```diff
@@ -20,6 +21,38 @@ export class FriendsRecommendationsList extends React.Component <IProps> {
...
+        this.props.changeUsersFollowing(_id, 'follow');
+    }
+
+    public unfollowRecommendationUser = (_id: string): void => {
+        this.props.changeUsersFollowing(_id, 'unFollow');
+    }
+
+    public dynamicButton = (_id: string, alreadyFollow: boolean): JSX.Element => {
+        if (this.props.loadFollow) {
+            return <span><Spinner color='light'/></span>;
```
Comment (anna-kuzii) [inline client/src/components/FriendsRecommendationsList/index.tsx]: do we need span here?

File: client/src/components/FriendsRecommendationsList/index.tsx [hunk 2]
```diff
@@ -20,6 +21,38 @@ export class FriendsRecommendationsList extends React.Component <IProps> {
...
+        }
+
+        if (alreadyFollow) {
+            return (
+                <span onClick={(): any => this.unfollowRecommendationUser(_id)}>
```
Comment (anna-kuzii) [inline client/src/components/FriendsRecommendationsList/index.tsx:35]: on click on span? Why not on Button? Do we need all these additional wrappers?

File: client/src/components/FriendsRecommendationsList/index.tsx [hunk 3]
```diff
@@ -20,6 +21,38 @@ export class FriendsRecommendationsList extends React.Component <IProps> {
...
+                        <Button className='btn' color='danger'>
+                            Unfollow
+                        </Button>
+                    </span>
+            );
+        }
+
+        return (
+            <span onClick={(): any => this.followRecommendationUser(_id)}>
```
Comment (anna-kuzii) [inline client/src/components/FriendsRecommendationsList/index.tsx:44]: same

File: client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx [hunk 4]
```diff
@@ -6,30 +6,35 @@ import {FriendsRecomendations} from './index';
 
 export const FriendsRecomendationsState = {
     loggedUsername: 'Maks',
+    loadFollow: false,
```
Comment (anna-kuzii) [inline client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx]: what the responsibility of this var?

File: client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx [hunk 5]
```diff
@@ -6,30 +6,35 @@ import {FriendsRecomendations} from './index';
...
     friendsRecommendations: {
         loading: false,
         users: [
             {
                 _id: '1f',
                 username: 'Diana',
                 photoPath: 'https://jsgram-post-images.s3.eu-west-2.amazonaws.com/1570605706200',
+                alreadyFollow: false,
```
Comment (anna-kuzii) [inline client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx]: bad name. Usually flag use with prefix is

**Review by anna-kuzii — CHANGES_REQUESTED** [review]

File: client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx (hunk 4 above)
Comment (IhorKalyta) [reply client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx]: No responsibility . Sorry i'm inattentive.

File: client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx (hunk 5 above)
Comment (IhorKalyta) [reply client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx]: fixed

File: client/src/store/newsFeed/tests/reducers.spec.ts [hunk 6]
```diff
@@ -116,18 +116,4 @@ describe('NewsFeed reducer test', () => {
             },
         });
     });
-
```
Comment (anna-kuzii) [inline client/src/store/newsFeed/tests/reducers.spec.ts]: why do u delete it?

**Review by anna-kuzii — CHANGES_REQUESTED** [review]

File: client/src/store/newsFeed/tests/reducers.spec.ts (hunk 6 above)
Comment (IhorKalyta) [reply client/src/store/newsFeed/tests/reducers.spec.ts]: add missed test

**Review by anna-kuzii — APPROVED** [review]

**Review by RevuraV — APPROVED** [review]

**Pull request merged**